frontend/src/views/AddAnimals/BeefCattle.tsx

```diff
@@ -19,16 +20,13 @@ export default function BeefCattle({
   formData: BeefCattleData;
 }) {
   const apiCache = useContext(APICacheContext);
-  const [, setSubtypes] = useState<BeefCattleSubtype[]>([]);
+  const [, Modal] = useState<BeefCattleSubtype[]>([]);
```

Comment:
How are we setting the subtypes but not reading them elsewhere?

Reply:
Moved the functionality of this BeefCattle file to a different BeefCattle file elsewhere.